integration-intyg-proxy-service/src/main/java/se/inera/intyg/minaintyg/integration/intygproxyservice/person/PersonConverterServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package se.inera.intyg.minaintyg.integration.intygproxyservice.person;
+
+import org.springframework.stereotype.Service;
+import se.inera.intyg.minaintyg.integration.api.person.Person;
+
+@Service
+public class PersonConverterServiceImpl implements PersonConverterService {
```

Comment:
I understand why some of the attributes stayed in Person. We are using the same class "...api.person.Person" to deserialise the result from intyg-proxy-service. Even if they are "the same", we should treat them as different classes. 

The class that we use for intyg-proxy-service could be named PersonDTO and need @Data annotation etc. But our Person-class could accually suffice with @Value and @Builder.